docs/running-on-yarn.md

```diff
@@ -54,8 +54,8 @@ In `yarn-cluster` mode, the driver runs on a different machine than the client,
 
 # Preparations
 
-Running Spark-on-YARN requires a binary distribution of Spark which is built with YARN support.
-Binary distributions can be downloaded from the Spark project website. 
+Running Spark on YARN requires a binary distribution of Spark which is built with YARN support.
```

Comment:
I see, it renders in code font already just not in the code box format. OK that seems fine. I think it's OK to merge this.

I'm referring to piecemeal edits since you've made a number of tiny PRs without a JIRA. It would be more efficient to have a review of whole logical sections of docs, and focus on small but not trivial changes in one PR. You'll probably end up spotting some broader opportunities to make docs consistent. Yes several of these changes here end up being good small fixes.